circuit/src/gadgets/and.rs

```diff
@@ -21,6 +21,17 @@ pub struct AndChip<F: FieldExt> {
 }
 
 impl<F: FieldExt> AndChip<F> {
+	/// Make a new chip.
+	/// # Examples
+	///
+	/// ```
+	/// let x = 1;
+	/// let y = 0;
+	/// // (x * y) - z == 0
+	/// //z is the next rotation cell for the x value.
+	/// let z = (x * y);
+	/// z;
+	/// ```
```

Comment:
We can move this example to the [line 56](https://github.com/lazovicff/eigen-trust/pull/81/files#diff-44a851e33abd6b4304994985eb6bc8d63d0154e82f1c43b436b1be6720198e39R56), and replace x and y with the actual values.

Reply:
Yes